src/apps/curve/ethereum/curve.pool.definitions.ts

```diff
@@ -129,6 +129,12 @@ export const CURVE_V1_POOL_DEFINITIONS: CurvePoolDefinition[] = [
     swapAddress: '0xfd5db7463a3ab53fd211b4af195c5bccc1a03890',
     tokenAddress: '0xfd5db7463a3ab53fd211b4af195c5bccc1a03890',
   },
+  {
+    queryKey: 'cadusd',
+    label: 'CADC-USDC Curve',
+    swapAddress: '0xe07bde9eb53deffa979dae36882014b758111a78',
+    tokenAddress: '0x1054ff2ffa34c055a13dcd9e0b4c0ca5b3aeceb9',
+  },
```

Comment:
The CADC-USDC pool is already supported since we support the Curve factory at [0xf18056bbd320e96a48e3fbf8bc061322531aac99](https://etherscan.io/address/0xf18056bbd320e96a48e3fbf8bc061322531aac99) that deployed this pool.

<img width="748" alt="image" src="https://user-images.githubusercontent.com/22452366/167277974-b8626adf-9274-4a8c-b482-8f248b533e4d.png">

As for the gauges, you can see [here](https://github.com/Zapper-fi/studio/blob/main/src/apps/curve/ethereum/curve.farm.contract-position-fetcher.ts#L157-L160) to see that we dynamically pull all gauge addresses from the factory contract at [0xb9fc157394af804a3578134a6585c0dc9cc990d4](https://etherscan.io/address/0xb9fc157394af804a3578134a6585c0dc9cc990d4). You can repeat the same logic for the factory at [0xf18056bbd320e96a48e3fbf8bc061322531aac99](https://etherscan.io/address/0xf18056bbd320e96a48e3fbf8bc061322531aac99).